e2e2/test/cases/nvidia/manifests/efa-device-plugin.yaml

```diff
@@ -24,63 +22,130 @@ spec:
+                    - c6id.metal
+                    - c6in.32xlarge
+                    - c6in.metal
+                    - c7a.48xlarge
+                    - c7a.metal-48xl
+                    - c7g.16xlarge
+                    - c7g.metal
+                    - c7gd.16xlarge
+                    - c7gn.16xlarge
+                    - c7i.48xlarge
+                    - c7i.metal-48xl
+                    - r5dn.24xlarge
+                    - r5dn.metal
+                    - r5n.24xlarge
+                    - r5n.metal
+                    - r6a.48xlarge
+                    - r6a.metal
+                    - r6i.32xlarge
+                    - r6i.metal
+                    - r6idn.32xlarge
+                    - r6idn.metal
+                    - r6in.32xlarge
+                    - r6in.metal
+                    - r6id.32xlarge
+                    - r6id.metal
+                    - r7a.48xlarge
+                    - r7a.metal-48xl
+                    - r7g.16xlarge
+                    - r7g.metal
+                    - r7gd.16xlarge
+                    - r7i.48xlarge
+                    - r7i.metal-48xl
+                    - r7iz.32xlarge
+                    - r7iz.metal-32xl
+                    - x2idn.32xlarge
+                    - x2idn.metal
+                    - x2iedn.32xlarge
+                    - x2iedn.metal
+                    - x2iezn.12xlarge
+                    - x2iezn.metal
+                    - i3en.12xlarge
+                    - i3en.24xlarge
+                    - i3en.metal
+                    - i4g.16xlarge
+                    - i4i.32xlarge
+                    - i4i.metal
+                    - im4gn.16xlarge
+                    - dl1.24xlarge
+                    - dl2q.24xlarge
+                    - g4dn.8xlarge
+                    - g4dn.12xlarge
+                    - g4dn.16xlarge
+                    - g4dn.metal
+                    - g5.8xlarge
+                    - g5.12xlarge
+                    - g5.16xlarge
+                    - g5.24xlarge
+                    - g5.48xlarge
+                    - inf1.24xlarge
+                    - p3dn.24xlarge
+                    - p4d.24xlarge
+                    - p4de.24xlarge
+                    - p5.48xlarge
+                    - trn1.32xlarge
+                    - trn1n.32xlarge
+                    - vt1.24xlarge
+                    - hpc6a.48xlarge
+                    - hpc6id.32xlarge
+                    - hpc7a.12xlarge
+                    - hpc7a.24xlarge
+                    - hpc7a.48xlarge
+                    - hpc7a.96xlarge
+                    - hpc7g.4xlarge
+                    - hpc7g.8xlarge
+                    - hpc7g.16xlarge
       hostNetwork: true
       containers:
-        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-efa-k8s-device-plugin:v0.3.3
-          imagePullPolicy: Always
+        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-efa-k8s-device-plugin:v0.5.0
```

Comment:
is there a latest tag for us to use so we can avoid keep updating version?

Reply:
Confirmed with EFA team, there is no latest tag.